internal/const/status/obtenantvariable/obtenantvariable.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright (c) 2023 OceanBase
+ob-operator is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+         http://license.coscl.org.cn/MulanPSL2
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details.
+*/
+
+package obtenantvariable
+
+const (
+	New       = "new"
+	PendingOB = "pending ob"
```

Comment:
What is the meaning of `pending ob`? It's somewhat confusing.

Reply:
This is initially used in obparameter status, pending ob means oceanbase cluster is under operation, this may happen when oceanbase cluster is upgrading, and the upgrade script will modify some parameters and the parameters are supposed not to be modified during the procedure, so we add this status for such scenarios.

Reply:
for variables, this status is not needed, I'll delete it